Libraries/LibWeb/Geolocation/GeolocationPositionError.cpp

```diff
@@ -25,4 +25,19 @@ void GeolocationPositionError::initialize(JS::Realm& realm)
     Base::initialize(realm);
 }
 
+// https://w3c.github.io/geolocation/#message-attribute
+FlyString GeolocationPositionError::message() const
```

Comment:
Does this need to be a FlyString? If we're matching against it, I suppose better to have the static ones to avoid initializing the FlyString each time on function call

Reply:
It does not!